app/src/main/java/sh/siava/AOSPMods/SettingsActivity.java

```diff
@@ -263,6 +263,7 @@ private void importExportSettings(boolean export) {
 		Intent fileIntent = new Intent();
 		fileIntent.setAction(export ? Intent.ACTION_CREATE_DOCUMENT : Intent.ACTION_GET_CONTENT);
 		fileIntent.setType("*/*");
+		fileIntent.putExtra(Intent.EXTRA_TITLE, "AOSPMods_Config" + ".bin");
```

Comment:
should the 
`+ ".bin"`
part be in line 265 for setType as the file extension that is needed?

Reply:
nope, all MIME type is here,https://www.iana.org/assignments/media-types/media-types.xhtml

and type: "*/*" will work,as I am lazy to find what type it should be